### ISSUE TITLE
regression(1.4.2=>devel): `nim c main` creates `pkgs` dir in current directory

### ISSUE THREAD

**Issue by timotheecour** [nim-lang/Nim#16556]
### Example
mkdir bar
touch bar/foo.nim
XDG_CONFIG_HOME= nim c bar/foo.nim

### Current Output
creates pkgs in cwd

### Expected Output
does not create pkgs in cwd

### Additional Information
devel 1.5.1 e869767aa72a71e673c2e8fdc51925f28c13d432
works in 1.4.2

**Comment by ringabout** [comment]
Do you USE choosenim?
see https://github.com/nim-lang/nimble/issues/864

**Comment by timotheecour** [comment]
yes,
which nim
/Users/timothee/.nimble//bin/nim

**Comment by timotheecour** [comment]
@genotrance https://github.com/nim-lang/nimble/issues/864#issuecomment-713007275
> This will be fixed in choosenim v0.7.4 which will be out soon.

however problem still there:
```
choosenim -v
choosenim v0.7.2 (2020-10-17 15:13:54) [macosx/amd64]

choosenim update self
   Updating choosenim
[##################################################] 100.0% 0kb/s
      Info: Updated choosenim to version 0.7.4

choosenim -v
choosenim v0.7.4 (2020-10-20 17:23:03) [macosx/amd64]

which nim
/Users/timothee/.nimble//bin/nim
XDG_CONFIG_HOME= nim c bar/foo.nim
```
=> still creates pkgs


## EDIT
updating choosenim wasn't enough, I had to also call this again: `choosenim /pathto/Nim`
```
choosenim /pathto/Nim
  Installed component 'nim'
    Prompt: Symlink for 'nimble' detected in '/Users/timothee/.nimble/bin'. Can I remove it? [y/N]
    Answer: n
  Installed component 'nimgrep'
  Installed component 'nimpretty'
  Installed component 'nimsuggest'
  Installed component 'testament'
  Installed component 'nim-gdb'
   Switched to Nim (/Users/timothee/git_clone/nim/Nim_devel)
```
after this, this bug is fixed
